pom.xml

```diff
@@ -3,11 +3,11 @@
   <groupId>com.couchbase</groupId>
   <artifactId>elasticsearch-transport-couchbase</artifactId>
   <version>2.1.1</version>
-  
+
     <properties>
-        <elasticsearch.version>1.7.1</elasticsearch.version>
+        <elasticsearch.version>2.2.0-SNAPSHOT</elasticsearch.version>
```

Comment:
Is there a release version for ES 2.x or 2.2.0?

Actually, I see that I logged this as a fix for "Can't install Plugin couchbase 4.0 + elasticsearch 2.0", but that is no longer the case.

It turns out that there are considerable differences in getting compatibility with ES 2.x vs ES 2.0.  I originally thought I could roll them both into the same PR, but I cannot.  This PR is actually to ensure compatibility with ES 2.x, as acquired by following the instructions to build the 2.x branch of the ES GitHub repo.  

I am still working on putting together a PR for the compatibility fix for ES 2.0.

Please advise.